projects/rocprofiler-systems/source/lib/rocprof-sys/library/sampling.cpp

```diff
@@ -1542,7 +1547,9 @@ post_process_perfetto(int64_t _tid, const std::vector<timer_sampling_data>& _tim
                                         auto _label = JOIN('-', "lineinfo", _n++);
                                         tracing::add_perfetto_annotation(
                                             ctx, _label.c_str(),
-                                            JOIN('@', demangle(line.name),
+                                            JOIN('@',
+                                                 rocprofsys::utility::get_demangler()
+                                                     .demangle(line.name),
```

Comment:
```suggestion
                                                 rocprofsys::utility::demangle(line.name),
```
Why do you use `rocprofsys::utility::demangle` in some cases and `rocprofsys::utility::get_demangler().demangle` in others?